base/Base/Base_Base_implementation.F90

```diff
@@ -3021,6 +3020,7 @@ module subroutine MAPL_GetHorzIJIndex(npts,II,JJ,lon,lat,lonR8,latR8,Grid, rc)
     real(ESMF_KIND_R8), allocatable :: target_lons(:),target_lats(:)
     real(ESMF_KIND_R8), allocatable :: corner_lons(:,:),corner_lats(:,:),center_lats(:,:),center_lons(:,:)
     type(ESMF_CoordSys_Flag) :: coordSys
+    character(len=ESMF_MAXSTR) :: grid_type
```

Comment:
It is not obvious to me how this change relates to OpenMP.

Reply:
This was introduced to get MAPL_GetHorzIJIndex to work correctly for mini grids. The original test using 
`if (im_world*6==jm_world) then`
failed for mini grids. Therefore a new test using 
`if(trim(grid_type) == "Cubed-Sphere") then`
was introduced.